spark/dl/src/test/scala/com/intel/analytics/bigdl/dataset/TransformersSpec.scala

```diff
@@ -458,6 +458,7 @@ class TransformersSpec extends FlatSpec with Matchers with BeforeAndAfter {
   }
 
   "RGBImage To SeqFile without file name" should "be good" in {
+    TestUtils.cancelOnWindows()
```

Comment:
hadoop will try to call chmod underhood and it will cause a failure. So cancel it on windows